### PR TITLE
change: [UIE-8194] DBaaS Settings maintenance pending updates tooltip should have accurate tooltip text

### PR DESCRIPTION
**Pull request by smans-akamai**
## Description 📝

This pull request is for a small change in the DBaaS Settings maintenance field. It updates the text used for the tooltip that gets displayed next to the Upgrade Version button when the button is disabled due to pending maintenance updates.

## Changes  🔄

- Text is changed to "Upgrades are disabled due to pending maintenance updates. To enable the upgrade, apply available updates now or wait until the next maintenance window."

## Target release date 🗓️

1/14/25

## Preview 📷

| Before  | After   |
| ------- | ------- |
| maintenance-tooltip-before | maintenance-tooltip-after-2 |

### Prerequisites

- Managed Databases
- dbaasV2 feature flag set for GA
- Use mock data so you can view the pending updates state reliably, or have a Default DB created e.g. postgresql 13 with pending updates available

### Verification steps
- [ ] For a new database cluster that has pending updates available. Access that Database Cluster Details and navigate to Settings.
- [ ] View the Maintenance field in Settings and there will be a tooltip visible next to the disabled Upgrade Version button.
- [ ] Hover over this field to view the updated text.

<details>
<summary> Author Checklists </summary>

## As an Author, to speed up the review process, I considered 🤔

👀 Doing a self review
❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
🤏 Splitting feature into small PRs
➕ Adding a [changeset](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md#writing-a-changeset)
🧪 Providing/improving test coverage
 🔐 Removing all sensitive information from the code and PR description
🚩 Using a feature flag to protect the release
👣 Providing comprehensive reproduction steps
📑 Providing or updating our documentation
🕛 Scheduling a pair reviewing session
📱 Providing mobile support
♿  Providing accessibility support

<br/>

- [x] I have read and considered all applicable items listed above.

## As an Author, before moving this PR from Draft to Open, I confirmed ✅

- [x] All unit tests are passing
- [x] TypeScript compilation succeeded without errors
- [x] Code passes all linting rules

</details>

### CODE REVIEW

File: packages/manager/src/features/Databases/DatabaseDetail/DatabaseSettings/DatabaseSettingsMaintenance.tsx [hunk 1]
```diff
@@ -52,7 +52,9 @@ export const DatabaseSettingsMaintenance = (props: Props) => {
             }}
             text={
               <Typography>
-                Upgrades are disabled while maintenance updates are in progress.
+                Upgrades are disabled due to pending maintenance updates. To
+                enable the upgade, apply available updates now or wait until the
```
Comment (mjac0bs) [inline packages/manager/src/features/Databases/DatabaseDetail/DatabaseSettings/DatabaseSettingsMaintenance.tsx:56]:
```suggestion
                enable the upgrade, apply available updates now or wait until the
```

**Review by mjac0bs — APPROVED** [review]
Approving pending typo is fixed and CI passes. Thanks @smans-akamai!

File: packages/manager/src/features/Databases/DatabaseDetail/DatabaseSettings/DatabaseSettingsMaintenance.tsx (hunk 1 above)
Comment (smans-akamai) [reply packages/manager/src/features/Databases/DatabaseDetail/DatabaseSettings/DatabaseSettingsMaintenance.tsx:56]: @mjac0bs Thanks for catching this. I'll push up a change

Comment (smans-akamai) [reply packages/manager/src/features/Databases/DatabaseDetail/DatabaseSettings/DatabaseSettingsMaintenance.tsx]: @mjac0bs I've pushed up a fix and changed the description for the PR.
That's what I get for copy pasting. Thanks again!

"Upgrades are disabled due to pending maintenance updates. To enable the upgrade, apply available updates now or wait until the next maintenance window."

**Review by mjac0bs — APPROVED** [review]

**Review by jdamore-linode — APPROVED** [review]

**Review by cpathipa — APPROVED** [review]

**Pull request merged**